targets/TARGET_STM/TARGET_STM32G0/STM32Cube_FW/STM32G0xx_HAL_Driver/stm32g0xx_ll_rtc.h

```diff
@@ -3074,7 +3074,7 @@ __STATIC_INLINE void LL_RTC_BKP_SetRegister(TAMP_TypeDef *TAMPx, uint32_t Backup
   */
 __STATIC_INLINE uint32_t LL_RTC_BKP_GetRegister(TAMP_TypeDef *TAMPx, uint32_t BackupRegister)
 {
-  const register __IO uint32_t *tmp;
+  register __IO uint32_t *tmp;
```

Comment:
Why?

Reply:
I was getting a build failure with an error that read something like this:

`ERROR: "register" not at beginning of declaration`

Reply:
Oops I guess it was a warning, not a build error.

Reply:
Reverted

Reply:
@jeromecoutant should this be in separate PR? the warning is pretty annoying